### PR TITLE
General UI Pull Request WIP

### PR DESCRIPTION
**Pull request by Maximus-**
Creating PR to monitor status of UI and track development.

### CODE REVIEW

File: iCepa.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -995,14 +1014,14 @@
...
+				DEVELOPMENT_TEAM = SS2V37KRFR;
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
 					"\"$(PROJECT_DIR)/Carthage/Build/iOS\"",
 				);
 				INFOPLIST_FILE = Extension/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @executable_path/../../Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = "$(CPA_EXTENSION_BUNDLE_IDENTIFIER)";
+				PRODUCT_BUNDLE_IDENTIFIER = pw.mxms000.tor.extension;
 				PRODUCT_NAME = iCepaTunnel;
```
Comment (FredericJacobs) [inline iCepa.xcodeproj/project.pbxproj]: Hey, 
Can you make sure to not change the bundle identifiers? In principle, if you just change it in the `xcconfig` and not commit those changes, it should build for you.

Comment (Maximus-) [inline iCepa.xcodeproj/project.pbxproj]: Will do

**Comment by conradev** [comment]
Thanks for opening this thread @Maximus-! Happy to start working on this and get some discussion going

**Comment by tladesignz** [comment]
Sorry, no. This is 3 years old and not working. Also, I have a full rewrite in progress. Won't merge, until working, but just to mention it here: https://github.com/tladesignz/iCepa/tree/restart

**Pull request closed without merging**